pkg/autodiscovery/common/utils/annotations_test.go

```diff
@@ -402,10 +403,40 @@ func TestBuildTemplates(t *testing.T) {
 				},
 			},
 		},
+		{
+			name:            "valid inputs with list and ignore ad tags",
```

Comment:
I think it would be best to not use abbreviations in the test name or struct fields. `ignoreAdTags`

What do you think about replacing `ad` for `auto discovery` ?

Reply:
I just realized that the k8s annotations are `ad.datadoghq.com/` 😆 . Maybe my comment is no that relevant then

Reply:
Thanks for the review ! 👍 
Indeed `ad` is a common abbreviation for autodiscovery.